### PR TITLE
Refresher_spec: Add deletions tests

### PR DESCRIPTION
**Pull request by zakiva**
Extracted k8s entities tests from https://github.com/ManageIQ/manageiq-providers-openshift/pull/18

@simon3z @cben @moolitayer Please review

### CODE REVIEW

File: spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb [hunk 1]
```diff
@@ -397,4 +397,103 @@ def label_with_name_value(name, value)
...
+      it "archives objects" do
+        expect(ContainerGroup.count).to eq(10)
+        expect(ContainerGroup.where(:deleted_on => nil).count).to eq(7)
+        expect(Container.count).to eq(10)
+        expect(Container.where(:deleted_on => nil).count).to eq(7)
+        expect(ContainerDefinition.count).to eq(10)
+        expect(ContainerDefinition.where(:deleted_on => nil).count).to eq(7)
+      end
+
+      it "removes the deleted objects from the DB" do
```
Comment (cben) [inline spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb]: I'd add here a comment with short overview of the deletions, to give _some_ context to the reader.
(deleted whole "my-project-0", many (all?) entities in "my-project-1", a pod and a route label in "my-project-2" — something like that)
Risks getting out sync.  Your call.

**Comment by zakiva** [comment]
> Eh, that's easier than I feared.
> The cassettes are exactly the same, right?

Yes, just copied them here

File: spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb (hunk 1 above)
Comment (zakiva) [inline spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb]: right, will do

**Comment by zakiva** [comment]
@miq-bot add_label test

File: spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb (hunk 1 above)
Comment (simon3z) [inline spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb]: @zakiva waiting for this before merging.

Comment (zakiva) [inline spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb]: done: https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/44/files#diff-75ab4d6ff092b098eabc70bba7571ca2R426

**Pull request merged**